### PR TITLE
Fixed envvar name for fn start and tweak to help text

### LINKED ISSUE #139: fn start should use `FN_LOG_LEVEL` rather than `GIN_MODE`
Mentioned most of this in https://github.com/fnproject/fn/issues/655 before closing it, but basically:

```
$ fn start --log-level DEBUG
can't create unix socket /var/run/docker.sock: device or resource busy
panic: gin mode unknown: DEBUG

goroutine 1 [running]:
github.com/fnproject/fn/vendor/github.com/gin-gonic/gin.SetMode(0xc420018029, 0x5)
	/go/src/github.com/fnproject/fn/vendor/github.com/gin-gonic/gin/mode.go:58 +0x156
..
```

```
"Env": [
                "GIN_MODE=DEBUG",
```

Help text is incorrect:

```
$ fn start --help
..
   --log-level value  --log-level DEBUG to enable debugging
..
```

And env var should be `FN_LOG_LEVEL`.

### PR DESCRIPTION
**Pull request by AstromechZA**
Fixes #139 and https://github.com/fnproject/fn/issues/655.

- help text was incorrect
- envvar passed to Docker container was incorrect

### CODE REVIEW

File: start.go [hunk 1]
```diff
@@ -19,14 +19,14 @@ func startCmd() cli.Command {
 		Flags: []cli.Flag{
 			cli.StringFlag{
 				Name:  "log-level",
-				Usage: "--log-level DEBUG to enable debugging",
+				Usage: "--log-level=debug to enable debugging",
```
Comment (treeder) [inline start.go]: Can we drop the equal sign here?

File: start.go [hunk 2]
```diff
@@ -46,9 +46,9 @@ func start(c *cli.Context) error {
 		"-p", "8080:8080",
 	}
 	if c.String("log-level") != "" {
-		args = append(args, "-e", fmt.Sprintf("GIN_MODE=%v", c.String("log-level")))
+		args = append(args, "-e", fmt.Sprintf("FN_LOG_LEVEL=%v", c.String("log-level")))
 	}
-	if c.String("env-file") != ""{
```
Comment (treeder) [inline start.go]: OT: @denismakogon looks like you're not running `go fmt`.

File: start.go (hunk 1 above)
Comment (AstromechZA) [reply start.go]: :+1: done

File: start.go (hunk 2 above)
Comment (denismakogon) [reply start.go]: we should put that into CI, but yeah, have fmt checks for fn repo in pre-commit hooks.

Comment (treeder) [reply start.go]: Regardless of CI, it looks like your fmt isn't working.

**Pull request merged**